### PR TITLE
Bugfixes for releasing ods-ci 1.19. Add AutomationBug and ProductBug

### PR DESCRIPTION
**Pull request by jgarciao**
- Fix tests checking image url for monitoring containers (now contain @sha256)
- Fix blackbox-exporter test updating expected number of endpoints 
- Try to fix flakyness when checking Grafana,Prometheus and BlackboxExpoter read…
- Fix Notebook CPU Usage test falkyness
- Add AutomationBug to "Verify Clean Up ODS Deployer"
- Add AutomationBug to "Verify Unauthorized User Is Not Able To Spawn Jupyter Notebook"
- Disable some Grafana tests for RHODS 1.20. Fix bug
- Fix some robocop issues

### CODE REVIEW

File: tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot [hunk 1]
```diff
@@ -117,24 +116,30 @@
...
     [Return]    ${cpu_usage}
-## TODO: Add this keyword with the other JupyterHub stuff
 
+CPU Usage Should Have Increased
+     [Documentation]   Verifies that CPU usage for notebook pods has increased since previous value
+     [Arguments]    ${cpu_usage_before}
+     ${cpu_usage_current} =    Read Current CPU Usage
+     Should Be True    ${cpu_usage_current}>${cpu_usage_before}
+
+# TODO: Add this keyword with the other JupyterHub stuff
```
[the remark this answers is not in the document]
Comment (jgarciao) [reply tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot:134]: Fixed

File: tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot [hunk 2]
```diff
@@ -117,24 +116,30 @@
...
 Run Jupyter Notebook For 5 Minutes
     Open Browser    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
     Login To RHODS Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
-    Wait for RHODS Dashboard to Load
+    Wait For RHODS Dashboard To Load
     Iterative Image Test    s2i-generic-data-science-notebook    https://github.com/lugi0/minimal-nb-image-test
     ...    minimal-nb-image-test/minimal-nb.ipynb
 
-##TODO: This is a copy of "Iterative Image Test" keyword from image-iteration.robob. We have to refactor the code not to duplicate this method
+# TODO: This is a copy of "Iterative Image Test" keyword from image-iteration.robob. We have to refactor the code not to duplicate this method
```
[the remark this answers is not in the document]
Comment (jgarciao) [reply tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot:142]: Fixed

File: tests/Tests/500__jupyterhub/test.robot [hunk 3]
```diff
@@ -25,10 +26,10 @@
~    #This keyword will work with accounts that are not cluster admins.
~    Launch Jupyterhub via App
~    Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
     Wait for RHODS Dashboard to Load
     Launch Jupyter From RHODS Dashboard Link
 
-Can Login to Jupyterhub
+Can Login To Jupyterhub
```
[the remark this answers is not in the document]
Comment (jgarciao) [reply tests/Tests/500__jupyterhub/test.robot:32]: Fixed

File: tests/Tests/500__jupyterhub/test.robot [hunk 4]
```diff
@@ -7,14 +7,15 @@
...
 Force Tags       JupyterHub
 
+
 *** Variables ***
 @{UNSUPPORTED_VAR_NAMES}=    1    invalid!    my_v@r_name    with space    L45t_0n3?!
 
 
 *** Test Cases ***
-Logged into OpenShift
+Logged Into OpenShift
```
[the remark this answers is not in the document]
Comment (jgarciao) [reply tests/Tests/500__jupyterhub/test.robot:18]: Fixed

**Comment by jgarciao** [comment]
Tested here:
- Smoke & Sanity excluding ProductBug, AutomationBug and  FlakyTest: [rhods-ci-pr-test/1386](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1386)
- All tests with ProductBug, AutomationBug and  FlakyTest: [rhods-ci-pr-test/1387](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1387)

Failed tests in Smoke & Sanity Tested again after implementing feedback from team:
- https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1391

**Review by bdattoma — APPROVED** [review]
lgtm

**Review by lugi0 — APPROVED** [review]

**Pull request merged**